### PR TITLE
Increase newman postman tests coverage

### LINKED ISSUE #392: Add consensus node calling endpoints to postman tests
### Problem

Currently the postman tests are missing calls that query the consensus nodes.
This would be useful in quick sanity checks to verify relay operation as they do not require knowledge of accounts and blocks details

### Solution

Add consensus  node calling checks
- `eth_gasPrice`
- `eth_feeHistory` - use 3 last blocks

Fix `base_url` vs `baseUrl` inconsistency between the `README.md` and `postman.js` files

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by Nana-EC**
**Description**:
Postman newman tests can be used to make multipl quick rpc calls to an endpoint.
The current solution is missing a few consensus node hitting calls

- Add `eth_gasPrice`
- Add `eth_feeHistory`
- Update `postman.json` to use `baseUrl`
- Add github action to run newman script

**Related issue(s)**:

Fixes #392 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/workflows/postman.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+on:
+  pull_request:
+    branches: [ main, release/** ]
+  push:
+    branches: [ main, release/** ]
+    tags: [ v* ]
+
+jobs:
+  setup-local-hedera:
+    name: Acceptance Tests
```
Comment (lukelee-sl) [inline .github/workflows/postman.yml:12]: I wonder if this should have a separate name?

Comment (Nana-EC) [reply .github/workflows/postman.yml:12]: Yes good, catch. Fixed it above missed this one

Comment (Nana-EC) [reply .github/workflows/postman.yml]: Fixed

**Comment by lukelee-sl** [comment]
LGTM

**Review by lukelee-sl — APPROVED** [review]

**Pull request merged**